cfgov/cfgov/settings/production.py

```diff
@@ -87,3 +88,9 @@
         }
 
 STATICFILES_STORAGE = 'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'
+
+# allow us to configure the default MySQL storage engine, via the environment
+if 'STORAGE_ENGINE' in os.environ:
+    db_options = {'init_command': os.environ['STORAGE_ENGINE']}
+    for db_label in DATABASES.keys():
+        DATABASES[db_label]['OPTIONS'] = db_options 
```

Comment:
I was thinking more like if someone was running `collectstatic` in production, which [uses a Sqlite DB](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/cfgov/settings/base.py#L185) for some reason.